cylc/rose/__init__.py

```diff
@@ -73,39 +73,28 @@
 Special Variables
 -----------------
 
-The Cylc Rose plugin has specific logic for handling a small group of special
-environment / template variables in the ``rose-suite.conf`` file:
+The Cylc Rose plugin provides two environment/template variables
+to the Cylc scheduler:
 
 ``ROSE_ORIG_HOST``
-   The plugin provides the hostname of the computer where the plugin runs as
-   an evironment variable.
+   The plugin provides the hostname of the computer where the plugin runs.
 
 ``ROSE_VERSION``
-   The plugin provides ``ROSE_VERSION`` from your installed Rose Version
-   in the environment section and any templating sections you have defined.
+   The plugin provides the version number of your installed Rose Version.
 
    .. deprecated:: 8.0.0
 
       Setting ``[env]ROSE_VERSION`` in ``rose-suite.conf``.
 
-      With Cylc 7 / Rose2019 users could set ``ROSE_VERSION`` for thier
+      With Cylc 7 / Rose2019 users could set ``ROSE_VERSION`` for their
       suites. This is no longer possible, and if set in your
-      ``ROSE_VERSION`` in your suite configuration it will be overwritten.
+      ``ROSE_VERSION`` in your suite configuration it will be ignored.
 
-   you set ``ROSE_VERSION`` in your ``rose-suite.conf`` it will be replaced.
 
-``CYLC_VERSION``
-   The plugin will remove ``CYLC_VERSION`` from your config as it is provided
-   by Cylc's config processing.
-
-   .. deprecated:: 8.0.0
-
-      Setting ``[env]CYLC_VERSION`` in ``rose-suite.conf``.
-
-      With Cylc 7 / Rose2019 users could set ``CYLC_VERSION`` for thier
-      suites. This is no longer possible, and if you set ``CYLC_VERSION``
-      in your suite configuration will be overwritten.
+.. caution::
```

Comment:
Can make this a deprecated item the same as for `ROSE_VERSION`.

Reply:
```suggestion
.. deprecated:: 8.0.0
```